lib/_stream_writable.js

```diff
@@ -458,6 +458,7 @@ function onwriteError(stream, state, er, cb) {
   --state.pendingcb;
 
   cb(er);
+  errorBuffer(state, new ERR_STREAM_DESTROYED('write'));
```

Comment:
@mcollina: I'm not sure what error to forward here... is `ERR_STREAM_DESTROYED` good enough? An alternative would be `er` but that would be a bit weird for writes that haven't actually been executed.

Reply:
Add a comment on why. I think this is correct.